pkg/reconciler/instances/istio/get/get_test.go

```diff
@@ -0,0 +1,87 @@
+package get
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"reflect"
+	"testing"
+)
+
+func TestGetIstioCRManifest(t *testing.T) {
```

Comment:
we are depending on network/http to execute unittests which is not nice. Can we mock this?